app/src/main/java/org/gem/indo/dooit/views/onboarding/LoginActivity.java

```diff
@@ -91,6 +91,15 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    protected void onResume() {
+        super.onResume();
+        Boolean wasRedirected = getIntent().getBooleanExtra("wasRedirected",false);
```

Comment:
Common Android practice is to put these magic strings in static fields.

```java
public static final String INTENT_WAS_REDIRECT = "was_redirect";
```

So the call becomes:

```java
getIntent().getBooleanExtra(INTENT_WAS_REDIRECT, false);
```

See `MinimalWebViewActivity`